src/plugins/dashboard/server/content_management/dashboard_storage.ts

```diff
@@ -341,7 +341,12 @@ export class DashboardStorage {
     const soResponse = await soClient.find<DashboardSavedObjectAttributes>(soQuery);
     const hits = soResponse.saved_objects
       .map((so) => {
-        const { item } = savedObjectToItem(so, false, soQuery.fields);
+        const { item } = savedObjectToItem(
+          so,
+          false,
+          soQuery.fields,
+          optionsToLatest?.includeReferences
```

Comment:
I am not sure I would continue with a list of parameters... but that's personal preference. I'd have an `options` object with an `includeReferences` prop.

Reply:
totally agree, will take look